modules/recommended-etcd-practices.adoc

```diff
@@ -25,7 +27,9 @@ The following hard disk features provide optimal etcd performance:
 * RAID 0 technology for increased performance.
 * Dedicated etcd drives. Do not place log files or other heavy workloads on etcd drives.
 
-Avoid NAS or SAN setups and spinning drives. Always benchmark by using utilities such as fio. Continuously monitor the cluster performance as it increases.
+Avoid NAS or SAN setups and spinning drives. Ceph RBD, as well as other network-attached storage, can result in unpredictable network latency. If possible, use PCI passthrough with NVM devices instead. 
+
+Always benchmark by using utilities such as fio. Continuously monitor the cluster performance as it increases. 
```

Comment:
Surround tool in backticks? (I see an example in the OCP guidelines doc)

> Always benchmark by using utilities such as `fio`. 

> Continuously monitor the cluster performance as it increases.

What is doing this monitoring, the tool or a user? I think the sentence needs a subject is missing. 


![Screenshot from 2023-06-09 14-43-43](https://github.com/openshift/openshift-docs/assets/57954076/2c460a63-a45f-4882-a4f6-a67d9d5c7ad9)

Reply:
"fio" is used as-is in the live docs already. I'm apt to leave it simply because I don't work in the scale and perf modules much. 

```suggestion
Always benchmark by using utilities such as fio. You can use such utilities to continuously monitor the cluster performance as it increases. 
```

I'm okay with the imperative statement, but also happy to be explicit.